steps/swift-environment-variable-injector/0.1.0/step.yml

```diff
@@ -0,0 +1,70 @@
+title: Swift Environment Variable Injector
+summary: |
+  Step to inject static values from environment variables into Swift code files before compilation and Continuous deployment allowing us to not expose our production keys and values in statically in our code and store sensitive values  safe.
+description: |
+  TThis step allows us to inject CI pipelines environment variables values into Swift code  static values before compilation and Continuous deployment to a specific environment(Development, Testing or Production) where we can define the values injected for each build and deployment e.g. a API url that the App uses and is different for each environment. Also it allows us to not expose our production keys and values in statically in our code.
+website: https://github.com/LucianoPAlmeida/bitrise-step-swift-envirionment-variable-injector
+source_code_url: https://github.com/LucianoPAlmeida/bitrise-step-swift-envirionment-variable-injector
+support_url: https://github.com/LucianoPAlmeida/bitrise-step-swift-envirionment-variable-injector/issues
+published_at: 2018-11-06T00:09:14.109021734-02:00
+source:
+  git: https://github.com/LucianoPAlmeida/bitrise-step-swift-envirionment-variable-injector.git
+  commit: 09112ef905f97aa5ea6a4c68722cae2e023ee4fe
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- macos
+type_tags:
+- dependency
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```

Comment:
does the step use all of these dependencies?
if not it takes unnecessary extra time to install them before the step run.